### PR TITLE
Added Tests for Profile Activity

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Added Tests for Profile Activity

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -81,15 +81,23 @@ dependencies {
...
     testImplementation "org.powermock:powermock-module-junit4:2.0.0-beta.5"
     testImplementation "org.powermock:powermock-api-mockito2:2.0.0-beta.5"
     testImplementation 'org.mockito:mockito-core:2.23.0'
     testImplementation "com.jraska.livedata:testing-ktx:1.1.2"
     testImplementation "androidx.arch.core:core-testing:2.1.0"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:5.3.1"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.3.1"
+
+    // Mockito and PowerMock
+    androidTestCompile ('org.powermock:powermock-mockito-release-full:1.6.0') {
```
Comment (domdomegg) [inline app/build.gradle:97]: Adding more powermock worries me a bit

File: app/build.gradle [hunk 2]
```diff
@@ -81,15 +81,23 @@ dependencies {
...
     testImplementation 'org.robolectric:robolectric:4.3'
-    testImplementation 'androidx.test:core:1.2.0'
+    testImplementation 'androidx.test:core:1.3.0'
     testImplementation "com.squareup.okhttp3:mockwebserver:$OKHTTP_VERSION"
     testImplementation "org.powermock:powermock-module-junit4:2.0.0-beta.5"
     testImplementation "org.powermock:powermock-api-mockito2:2.0.0-beta.5"
     testImplementation 'org.mockito:mockito-core:2.23.0'
     testImplementation "com.jraska.livedata:testing-ktx:1.1.2"
     testImplementation "androidx.arch.core:core-testing:2.1.0"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:5.3.1"
```
Comment (domdomegg) [inline app/build.gradle:93]: Good we're moving over to JUnit 5 - would be good to get all the tests onto it. As a test-only dependency it won't have any affect on the final app so should be fine.

**Review by domdomegg — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (madhurgupta10) [reply app/build.gradle:97]: @domdomegg I wasn't sure how to test the static method without adding this import, let me know if there is a better way :)

Comment (domdomegg) [reply app/build.gradle:97]: Makes sense :)

In an ideal world we'd avoid these static methods, but appreciate that might not be possible here. Still very happy for this to be merged (more tests are almost always good in my eyes), but just over time it'd be nice if we could reduce our usage of powermock.

I think these posts explain my opinions on powermock, but happy to have a wider discussion too :)
https://blog.frankel.ch/on-powermock-abuse/
https://stackoverflow.com/a/30163045/5352227
https://medium.com/@inderivita/powermock-testing-tool-or-testing-obstacle-29f3303c9337

**Pull request merged**